test/e2e/metrics_test.go

```diff
@@ -1,4 +1,4 @@
-// +build e2e
+// +build cli
```

Comment:
This doesn't seem to be a `cli` test?

Reply:
good point - I've moved the shared code to new files and reverted this